packages/react-components/react-carousel/library/src/components/useEmblaCarousel.ts

```diff
@@ -196,6 +214,7 @@ export function useEmblaCarousel(
           emblaApi.current?.on('reInit', handleReinit);
           emblaApi.current?.on('slidesInView', handleVisibilityChange);
           emblaApi.current?.on('select', handleIndexChange);
+          emblaApi.current?.on('slidesChanged', handleSlideChange);
         }
       },
     };
```

Comment:
@Mitch-At-Work as you are starting to fix bugs, please also add a E2E (Cypress) test for this scenario.